addons/website_sale/__manifest__.py

```diff
@@ -10,7 +10,7 @@
 ==================
 
         """,
-    'depends': ['website', 'sale', 'payment', 'website_payment', 'website_portal_sale', 'website_mail', 'website_form', 'rating'],
+    'depends': ['website', 'sale', 'account', 'payment', 'website_payment', 'website_portal_sale', 'website_mail', 'website_form', 'rating'],
```

Comment:
I thank it was good because there was already an indirect dependancy: website_sale depends on sale and sale depends on account. So, I'm writing a bridge-module.